frontend/generated/gql/fragment-masking.ts

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
```

Comment:
In DIAPP the `generated` directory is added to `.gitignore`, so I think it is not necessary to add these files to git.